tasks/build_tags.py

```diff
@@ -274,13 +251,16 @@
 }
 
 # List of tags to always remove when building on Darwin/macOS
-DARWIN_EXCLUDED_TAGS = {"docker", "containerd", "no_dynamic_plugins", "nvml", "cri", "crio"}
+DARWIN_EXCLUDED_TAGS = {"docker", "containerd", "nvml", "cri", "crio"}
 
 # Unit test build tags
 UNIT_TEST_TAGS = {"test"}
 
 # List of tags to always remove when running unit tests
-UNIT_TEST_EXCLUDE_TAGS = {"datadog.no_waf", "pcap"}
+UNIT_TEST_EXCLUDE_TAGS = {"pcap"}
+
+# Common build tags, added on all builds
+COMMON_TAGS = {"grpcnotrace", "no_dynamic_plugins", "trivy_no_javadb", "datadog.no_waf"}
```

Comment:
do we have supporting docs (public or internal) that we can link to that explain why we use each of these?

Reply:
We have detailed comments for `grpcnotrace` and `no_dynamic_plugins` where they are defined at the top of this file
`trivy_no_javadb` and `datadog.no_waf` don't have such a comment though, I can add one